packages/ui/src/utils/utils.ts

```diff
@@ -110,3 +110,34 @@ export function formatNumber(num: number | bigint): string {
 export interface Violation {
   violation: string
 }
+
+/**
+ * Convert nanoseconds to simple human-readable format
+ * @example
+ * formatNs(416_328_583) // "416ms"
+ * formatNs(1_234_567_890) // "1s 234ms"
+ * formatNs(90_000_000_000) // "1m 30s"
+ */
+export function formatNs(nanoseconds: number): string {
```

Comment:
we already have a function that takes milliseconds and returns this same format.
can we just extend/reuse that function to accept nanoseconds as well?